### DISCUSSION TITLE
Once min

### DISCUSSION THREAD

**Discussion by mlin2017** [GreenleafLab/ArchR#1965] [Questions / Documentation]
Hi,

Thanks for this amazing tool! 

After running CreateArrowFiles(), I noticed that on QC plots of Unique nuclear fragments vs TSS enrichment scores, there are no cells with fragments less than the minFrags (left side of the vertical minFrags line), while cells with TSS scores less than minTSS exist (below the horizontal minTSS line). To figure out at which step cells got filtered out, I read through the CreateArrow(), and  cells with fragments less than minFrags got filtered out at tmpToArrow() step, before calculating TSS scores. Could anyone confirm that I got it correctly? If so, does that mean that if we set a relatively high minFrags threshold at the beginning, we will not be able to recover cells with smaller number of fragments than this cutoff unless we re-run from the very beginning with a smaller minFrags?

However, in the tutorial (https://www.archrproject.com/bookdown/per-cell-quality-control.html), CD34 BMMC and PBMC example datasets do have dots on the left side of the minFrags threshold. Is this due to code changes between different versions (I am using v1.0.2)? 

Thanks!

**Comment by rcorces** [comment]
all of that is correct. cells are filtered before TSS to make that go quicker. the plots in the tutorial are very old.
